### ISSUE TITLE
Ability to run ng build --watch for multiple projects in parallel

### ISSUE THREAD

**Issue by Roaders** [angular/angular-cli#13373]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ x] feature request
```

### Command (mark with an `x`)
```
- [ ] new
- [x ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Versions

Angular CLI: 7.1.4
Node: 10.14.1
OS: win32 x64
Angular: 7.1.4
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.11.4
@angular-devkit/build-angular     0.11.4
@angular-devkit/build-optimizer   0.11.4
@angular-devkit/build-webpack     0.11.4
@angular-devkit/core              7.1.4
@angular-devkit/schematics        7.1.4
@ngtools/webpack                  7.1.4
@schematics/angular               7.1.4
@schematics/update                0.11.4
rxjs                              6.3.3
typescript                        3.1.6
webpack                           4.23.1

### Desired functionality
I am currently building an electron app and chrome extension. Both these apps are built together and interact with each other so development of both at the same time makes sense.
Because of this I have 3 builds:

* Angular Electron Renderer App
* Angular Chrome Extension App
* Electron main (node)

I am setting up an angular project for each of these but I really want them all built at the same time. In webpack I simply add more input files and have 1 webpack build that watches and compiles all 3. 

Is there an equivalent approach in angular?

**Comment by alan-agius4** [comment]
Duplicate of: 
https://github.com/angular/angular-cli/issues/11002
